src/documents/StandardizedTestTools.md

```diff
@@ -90,30 +112,37 @@ This item requires  examination on a given topic requiring a written analysis or
-<li> Students who rely on text-to-speech software require longer time to digest information and recognize the subtle distinctions between choices. A dyslexic student may be unable to identify such subtle distinctions at all.</li>
-<li> Questions that address higher-order understanding rather than surface learning are very complex and can require numerous re-readings. This has the potential to disadvantage the visually impaired student and those using text-to-speech software.</li>
-<li> Progressing between options using the tab key (for students who cannot use a mouse) can take up a great deal of additional time.</li>
+<li> For visually-impaired students who rely on text-to-speech software, remembering a variety of spoken information can
+require considerable mental effort.</li>
+<li> Students who rely on text-to-speech software require longer time to digest information and recognize the subtle
+distinctions between choices. A dyslexic student may be unable to identify such subtle distinctions at all.</li>
+<li> Questions that address higher-order understanding rather than surface learning are very complex and can require
+numerous re-readings. This has the potential to disadvantage the visually impaired student and those using
+text-to-speech software.</li>
+<li> Progressing between options using the tab key (for students who cannot use a mouse) can take up a great deal of
+additional time.</li>
 </ul>
 </td>
 <td>
 <ul>
 <li> Provide a printed version or a braille transcription (if applicable).</li>
-<li> Provide different methods for selecting a response, such as mouse click, keyboard, touch screen, speech recognition, assistive devices to access the keyboard and select (e.g., mouth stick, head wand, one-and two-switch systems).</li>
+<li> Provide different methods for selecting a response, such as mouse click, keyboard, touch screen, speech
+recognition, assistive devices to access the keyboard and select (e.g., mouth stick, head wand, one-and two-switch systems).</li>
 <li> Provide an option for paper/pencil in addition to computer (e.g., scratch paper for solving problems, drafting ideas).</li>
 <li> Try to avoid overly-complex answers, especially long lists with distinctions that can only be made by careful re-readings.</li>
-<li> If using graphics, ensure that high-quality recorded descriptions are available for each graphic used or that a narrative is included with video clips to describe any action taking place.</li>
+<li> If using graphics, ensure that high-quality recorded descriptions are available for each graphic used or that a
+narrative is included with video clips to describe any action taking place.</li>
 </ul>
 </td>
 </tr>
 <tr>
-<td>** *Labeling Questions* **,
-** *Grouping and Ordering* **,
-** *Interactive Questions* **
+<td>***Labeling Questions***,
+***Grouping and Ordering***,
+***Interactive Questions***
```

Comment:
There is something wrong with basically everything in this column. I think what the intention may have been was to have things both **bold** and _italicized_. However, they are inside of HTML markup, so we'd have to actually do this with HTML and not Markdown. Maybe these could be `<th>` but not sure if that makes sense in this context.

Reply:
Addressed in #40.